pkg/store/fsm.go

```diff
@@ -88,11 +93,24 @@ func (s *Store) Apply(l *raft.Log) (e interface{}) {
 		}
 		if e, ok := s.enforcers.Load(cmd.Namespace); ok {
 			enforcer := e.(*casbin.DistributedEnforcer)
+			a, err := adapter.NewAdapter(s.enforcersState, cmd.Namespace, "")
+			if err != nil {
+				return &FSMResponse{error: err}
+			}
 			model, err := model2.NewModelFromString(p.Text)
 			if err != nil {
 				return &FSMResponse{error: err}
 			}
-			enforcer.SetModel(model)
+			// load existed policies
+			err = a.LoadPolicy(model)
+			if err != nil {
+				return &FSMResponse{error: err}
+			}
+			err = enforcer.InitWithModelAndAdapter(model, a)
```

Comment:
`InitWithModelAndAdapter` includes `LoadPolicy` and `BuildRoleLinks`, so don't call that.